package.json

```
@@ -59,6 +59,7 @@
     "eslint": "^8.35.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.6.1",
     "eslint-plugin-node": "^11.1.0",
```

We should remove `eslint-plugin-node` in favor of `eslint-plugin-n` as the former is no maintained.

---

Do you have a link that references that?

---

I don't have a direct link referencing that but `eslint-plugin-node` hasn't been updated in 3 years, and `eslint-plugin-n` also mentions it at the very top of their README, which references this issue: https://github.com/mysticatea/eslint-plugin-node/issues/300

---

I agree it doesn't make sense to keep both.
```suggestion
    "eslint-plugin-n": "^15.6.1",
```